mantidproject/mantid
# Update testing guide for MuonAnalysis

~~**Do not merge until #18013 is merged**~~

Updated the unscripted testing guide for Muon Analysis to incorporate some important points:
- Switching between old and new style UI
- "Fit to raw data" option
- Auto-setting the fit label

Since this made the original guide very long and unwieldy, I split it into three sections and added more screenshots.

**To test:**

*Documentation only - no code changes*
Check that the docs build OK and all links, images display correctly. The testing instructions should make sense.

Fixes #17618.

*No release notes - for internal use*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
